pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdBrokers.java

```diff
@@ -146,6 +146,15 @@ void run() throws Exception {
 
     }
 
+    @Parameters(commandDescription = "Get the version of the currently connected broker")
+    private class PulsarVersion extends CliCommand {
+
+        @Override
+        void run() throws Exception {
+            System.out.println( getAdmin().brokers().getVersion());
```

Comment:
I believe that we have a little usability problem here.

The user does not know which broker reported the version.

Proposals:
- return the address of the broker that reported the version (a BrokerInfo structure ?)
- add a (optional) parameter to specify the id of the broker that we want to query

In a Pulsar cluster usually you have the same version of the software, but this may not be the case during upgrades or in case you have special versions of Pulsar on some nodes.

Reply:
Thanks, I agree with you, but I think the current version of the broker can be queried explicitly using the `pulsar-admin brokers version` command, for example, the `pulsar-admin brokers version` command will use the `webServiceUrl` configuration in the `client.conf` configuration file, and of course, the `webServiceUrl` parameter can be used to specify the address of the broker to be queried by the command `pulsar-admin --admin-url`, In a pulsar cluster upgrade, multiple versions of the broker will exist at the same time, and with the `--admin-url` parameter, we can explicitly get the version of the specified broker